tests/01_unit/experiments/test_atmosphere.py

```diff
@@ -166,6 +166,7 @@ def test_atmosphere_experiment_real_life(
     assert mi_sensors["radiancemeter"].medium().id() == "medium_atmosphere"
 
 
+@pytest.mark.slow
```

Comment:
We need to review the performance of these tests and remove the mark if applicable.